integration_tests/mockApis/referrals.ts

```diff
@@ -31,6 +38,39 @@ export default {
       },
     }),
 
+  stubReferralWithScenario: ({
+    newScenarioState,
+    referral,
+    requiredScenarioState,
+    scenarioName,
+  }: ReferralAndScenarioOptions): SuperAgentRequest =>
```

Comment:
Is there a way we can explain what these mean/do? The Wiremock documentation is awful